cmd/mysterium_fake/mysterium_fake.go

```diff
@@ -20,7 +20,22 @@ func main() {
 	waiter := &sync.WaitGroup{}
 	mysteriumClient := server.NewClientFake()
 
-	serverCommand := command_server.NewCommandWith(
+	var serverCommand *command_server.Command
+	var clientCommand *command_client.Command
+
+	stop := func() {
+		var killers []cmd.Killer
+		if serverCommand != nil {
+			killers = append(killers, serverCommand.Kill)
+		}
+		if clientCommand != nil {
+			killers = append(killers, serverCommand.Kill)
+		}
+		stop := cmd.NewApplicationStopper(killers...)
```

Comment:
Array of callbacks looks overcomplicated

Reply:
Simplified.